codec-agnostic-rtp-payload-format.md

```diff
@@ -43,27 +43,37 @@ informative:
   RFC6464:
   RFC6465:
   RFC6904:
+  RFC6184:
   SFrame:
+    target: https://tools.ietf.org/html/draft-omara-sframe
+    title: Secure Frame (SFrame)
+  WebRTCInsertableStreams:
+    target: https://w3c.github.io/webrtc-insertable-streams
+    title: WebRTC Insertable Media using Streams
 
 --- abstract
 
-
+WebRTC Media Chains usually rely on piping encoder output directly to packetizers. Media packetization formats often support a specific codec format and optimize RTP packets generation accordingly.
+With the development of Selective Forward Unit (SFU) solutions, WebRTC Media Chains are increasingly relying on application-specific transforms that seat between encoder and packetizer on one end and between depacketizer and decoder on the other end. These transforms are typically protecting media content from being accessible from the SFU, for instance using {{SFrame}} or {{WebRTCInsertableStreams}}.
```

Comment:
I think we should focus on Sframe and media encryption as main drivers for the generic packetizer and then abstract the requirement to a Media Transform, if not we could be challenged that those transforms should be specified and done in an "RTP standard way"

Reply:
I tried to do that here: this sentence is using "protecting media content from being accessible from the SFU" to explain what the main driver is.
I would think that WebRTCInsertableStreams, in its form, forbids using the "RTP standard way".
Any suggestion on your side to improve the sentence?

Reply:
RTP Media Chains usually rely on piping encoder output directly to packetizers. Media packetization formats often support a specific codec format and optimize RTP packets generation accordingly.

However the need for providing higher layers of security, some implementation are increasingly relying on application-specific transforms for performing media encryption and decryption that seat between encoder and packetizer on one end and between depacketizer and decoder on the other end, for instance using {{SFrame}} or {{WebRTCInsertableStreams}}.

Reply:
Ah, that looks good too. I went with a slightly different wording but I am ok with using your version instead, let me know.